src/style.css

```diff
@@ -12,15 +12,17 @@
 .rct9k-items-outer {
   white-space: nowrap;
   position: absolute;
-  display: inline-block;
+  display: flex;
+  overflow: visible;
```

Comment:
This css is included by demo.js. But is it included in the build of the lib? So that an user app can reuse it? Or is it done by copying.

If reuse case: I'm wondering about the backwards compatibility for people that reused it. I cannot project the modifications that I see in the file. But my gut feeling tells me that they may be breaking changes for some?